source/_components/homematicip_cloud.markdown

```diff
@@ -28,6 +29,7 @@ redirect_from:
   - /components/light.homematicip_cloud/
   - /components/sensor.homematicip_cloud/
   - /components/switch.homematicip_cloud/
+  - /components/weather.homematicip_cloud/
```

Comment:
this is not necessary

Reply:
Removed line 20 and 32

Reply:
You can keep line 20 but the redirects are for existing pages.